src/field/pokemon.ts

```diff
@@ -1747,7 +1747,10 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
         power.value = 60;
       }
       applyPreAttackAbAttrs(VariableMovePowerAbAttr, source, this, move, power);
-      this.scene.getField(true).map(p => applyPreAttackAbAttrs(FieldVariableMovePowerAbAttr, this, source, move, power));
+      const fieldAuraPokemon = this.scene.getField(true).find(p => p.hasAbilityWithAttr(FieldVariableMovePowerAbAttr));
+      if (fieldAuraPokemon) {
+        applyPreAttackAbAttrs(FieldVariableMovePowerAbAttr, fieldAuraPokemon, fieldAuraPokemon, move, power);
+      }
```

Comment:
wont this be enough? 

```suggestion
      this.scene.getField(true).map(p => applyPreAttackAbAttrs(FieldVariableMovePowerAbAttr, p, this, move, power));
```

Reply:
Doing this would make the damage bonus from the ability apply for every pokemon on the field that has the aura ability.
If two pokemon have dark_aura then the damage bonus would apply twice. I didn't think that was desirable behavior so I stop after the first is found.

Reply:
> Doing this would make the damage bonus from the ability apply for every pokemon on the field that has the aura ability. If two pokemon on the field have dark_aura then the damage bonus would apply twice. I didn't think that was desirable behavior so I stop after the first is found.

i see, i did not think of that. nice catch! but why are you passing the same pokemon on the `applyPreAttackAbAttrs`?

Reply:
it doesn't matter which you pass in as the second 'defender' pokemon since `FieldVariableMovePowerAbAttr` doesn't use the defender at all. Ideally in the future we would get rid of this unnecessary variable passing but there is a PR blocking me from doing that

Reply:
gotcha, consider this resolved, thanks!